### PR TITLE
Add tests for guest user and logged in user viewing category drop-down

### PR DESCRIPTION
**Pull request by cmacaulay**
👍 
- All tests still passing

### CODE REVIEW

File: spec/features/nav_footer/user_sees_nav_and_footer_spec.rb [hunk 1]
```diff
@@ -19,6 +19,40 @@
...
+    visit root_path
+
+    within("nav") do
+      expect(page).to have_link("Products")
+    end
+
+    within(".dropdown-content") do
+      expect(page).to have_link(category.name)
+      expect(page).to have_link(category_2.name)
+    end
```
Comment (meyerhoferc) [inline spec/features/nav_footer/user_sees_nav_and_footer_spec.rb:34]: 👍

File: spec/features/nav_footer/user_sees_nav_and_footer_spec.rb [hunk 2]
```diff
@@ -19,6 +19,40 @@
...
```
Comment (meyerhoferc) [inline spec/features/nav_footer/user_sees_nav_and_footer_spec.rb:53]: 👍

**Review by meyerhoferc — APPROVED** [review]
glad we have tests for these now, ready for second review & merge @CjMoore @valerietrudell

File: spec/features/nav_footer/user_can_navigate_site_using_nav_and_footer_spec.rb [hunk 3]
```diff
@@ -0,0 +1,125 @@
+require 'rails_helper'
+
```
Comment (CjMoore) [inline spec/features/nav_footer/user_can_navigate_site_using_nav_and_footer_spec.rb:2]: Super thorough tests. Thanks.

**Review by CjMoore — APPROVED** [review]
Looks great merging now!

**Pull request merged**